python/examples/dspy-rag-fastapi/README.md

```diff
@@ -55,6 +55,7 @@ ENVIRONMENT=<your_environment_value>
 INSTRUMENT_DSPY=<true or false>
 COLLECTOR_ENDPOINT=<your_arize_phoenix_endpoint>
 OPENAI_API_KEY=<your_openai_api_key>
+CO_API_KEY=<your_cohere_api_key>
```

Comment:
Are both necessary, or only one? Would be helpful to clarify.

Reply:
I made it semi-optional. You don't get to use cohere if you don't add the key